<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>列车售票管理系统--admin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>欢迎使用列车售票管理系统！</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="3">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="6">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="TrainBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>  班次管理  </string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="FileBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>数据读写</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="BuyBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>乘客购票</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="ReturnBtn">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>乘客查询/退票</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="SearchBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>  查询班次  </string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="PassengerBtn">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>乘客管理</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>班次乘客管理</string>
    </property>
    <addaction name="actionTrain"/>
    <addaction name="actionPassengers"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>购票/退票/乘客查询</string>
    </property>
    <addaction name="actionBuy"/>
    <addaction name="actionReturn"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查询/数据读写</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionFile"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>账号管理</string>
    </property>
    <widget class="QMenu" name="menuAccount">
     <property name="title">
      <string>账号</string>
     </property>
     <addaction name="actionShow"/>
     <addaction name="separator"/>
     <addaction name="actionChangepsw"/>
    </widget>
    <addaction name="menuAccount"/>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menu_5"/>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTrain">
   <property name="text">
    <string>班次管理</string>
   </property>
  </action>
  <action name="actionPassengers">
   <property name="text">
    <string>乘客管理</string>
   </property>
  </action>
  <action name="actionBuy">
   <property name="text">
    <string>购票</string>
   </property>
  </action>
  <action name="actionReturn">
   <property name="text">
    <string>退票/乘客查询</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>查询数据</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>列车售票管理系统</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionTws">
   <property name="text">
    <string>tws</string>
   </property>
  </action>
  <action name="actionNewAccount">
   <property name="text">
    <string>新建管理账户</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>退出登录</string>
   </property>
  </action>
  <action name="actionRead">
   <property name="text">
    <string>读取文件</string>
   </property>
  </action>
  <action name="actionWrite">
   <property name="text">
    <string>输出文件</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>show</string>
   </property>
  </action>
  <action name="actionChangepsw">
   <property name="text">
    <string>修改密码</string>
   </property>
  </action>
  <action name="actionUseraccount_2">
   <property name="text">
    <string>用户管理</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>数据读写</string>
   </property>
  </action>
  <action name="actionUsers">
   <property name="text">
    <string>用户管理</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
